front/src/pages/songs/[slugOrId]/[...tab].tsx

```diff
@@ -179,6 +180,25 @@ const SongPage: Page<GetPropsTypesFrom<typeof prepareSSR>> = ({ props }) => {
 								song.data && `${song.data?.name} (${t("info")})`
 							}
 						/>
+						{song.data?.type && (
```

Comment:
Could you do something similar to the BPM? I.e. When the data is loading, show a skeleton.
The `Song Type` label should always appear though. Something like:

```ts
// Pseudo code
<Stack row>
  <Typography>{songType}</Typopgraphy>
  {song.data ? <Stack row>{/*show icon and song type*/}</Stack> : <Skeleton />}
</Stack>
```